File: spring/src/main/java/bitc/full502/spring/dto/LodgingBookingDto.java
package bitc.full502.spring.dto;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class LodgingBookingDto {
    private int adult;
    private int child;
    private String ckIn;
    private String ckOut;
    private Long lodId;
    private String roomType;
    private String status;
    private Long totalPrice;
    private Long userId;
}

File: android_studio/app/src/main/java/bitc/fullstack502/android_studio/network/dto/LodgingBookingDto.kt
package bitc.fullstack502.android_studio.network.dto

data class LodgingBookingDto(
    val userId: Long,
    val lodId: Long,
    val ckIn: String,
    val ckOut: String,
    val totalPrice: Long,
    val roomType: String,
    val adult: Int,
    val child: Int,
    val status: String
)
